doc/src/index.md

```diff
@@ -1,4 +1,15 @@
-# Julia Documentation
+# Julia v0.6 Documentation
+
+Welcome to the documentation for Julia 0.6!
+
+### What's new in Julia 0.6
+
+Please see the [Release notes](NEWS.md) for a list of new features, changes and deprecations from 0.5 to 0.6.
+
+For more in detail analysis of some of the new functionalities in 0.6, see the following blog posts:
```

Comment:
Avoiding "0.6's", perhaps "For a closer look at some of this release's new features, see..." or "For a closer look at [certain|specific|particular] new features in 0.6, see..."?